jmix-flowui/flowui-kit/src/main/java/io/jmix/flowui/kit/component/main/ListMenu.java

```diff
@@ -326,6 +320,14 @@ protected RouterLink createMenuItemComponent(MenuItem menuItem) {
         return routerLink;
     }
 
+    protected void addMenuOpenedChangeListener(Details details, MenuItem menuItem) {
+        details.addOpenedChangeListener((ComponentEventListener<Details.OpenedChangeEvent>) event -> {
```

Comment:
You add listener to `details` for every `menuItem`, but you can have a single listener and iterate over all menu items and invoke their `OpenedChangeHandler`.